config/config.go

```diff
@@ -988,7 +1005,28 @@ func SetupViper(v *viper.Viper, filename string) {
 	// Migrate config settings to maintain compatibility with old configs
 	migrateConfig(v)
 	migrateConfigPurposeOneTreatment(v)
+	migrateConfigTCF2PurposeEnabledFlags(v)
 
+	v.SetDefault("gdpr.tcf2.purpose1.enabled", true)
+	v.SetDefault("gdpr.tcf2.purpose2.enabled", true)
+	v.SetDefault("gdpr.tcf2.purpose3.enabled", true)
+	v.SetDefault("gdpr.tcf2.purpose4.enabled", true)
+	v.SetDefault("gdpr.tcf2.purpose5.enabled", true)
+	v.SetDefault("gdpr.tcf2.purpose6.enabled", true)
+	v.SetDefault("gdpr.tcf2.purpose7.enabled", true)
+	v.SetDefault("gdpr.tcf2.purpose8.enabled", true)
+	v.SetDefault("gdpr.tcf2.purpose9.enabled", true)
+	v.SetDefault("gdpr.tcf2.purpose10.enabled", true)
+	v.SetDefault("gdpr.tcf2.purpose1.enforce_purpose", "full")
+	v.SetDefault("gdpr.tcf2.purpose2.enforce_purpose", "full")
+	v.SetDefault("gdpr.tcf2.purpose3.enforce_purpose", "full")
+	v.SetDefault("gdpr.tcf2.purpose4.enforce_purpose", "full")
+	v.SetDefault("gdpr.tcf2.purpose5.enforce_purpose", "full")
+	v.SetDefault("gdpr.tcf2.purpose6.enforce_purpose", "full")
+	v.SetDefault("gdpr.tcf2.purpose7.enforce_purpose", "full")
+	v.SetDefault("gdpr.tcf2.purpose8.enforce_purpose", "full")
+	v.SetDefault("gdpr.tcf2.purpose9.enforce_purpose", "full")
+	v.SetDefault("gdpr.tcf2.purpose10.enforce_purpose", "full")
```

Comment:
Why are these values not grouped with the other tcf2 values?

Reply:
Gotcha. I did not understand the significance of the placement. Could you please add a comment explaining this situation?